src/shogun/clustering/KMeansBase.h

```
@@ -76,6 +76,12 @@ class KMeansBase : public RandomMixin<DistanceMachine>
 		 */
 		SGMatrix<float64_t> get_cluster_centers() const;
 
+		/** get cluster standard deviations
+		 *
+		 * @return cluster centers or empty matrix if no radiuses are there (not trained yet)
```

if not trained yet, it should throw an error